streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -487,7 +506,11 @@ public void advanceWallClockTime(final long advanceMs) {
      * @see #getSessionStore(String)
      */
     public StateStore getStateStore(final String name) {
-        return ((ProcessorContextImpl) task.context()).getStateMgr().getStore(name);
+        StateStore res = task == null ? null : 
```

Comment:
Ditto, could `task` field be null here?

Reply:
`res` -> `stateStore`

Reply:
Yes it could be if there are only global tables.